site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -112,14 +131,42 @@ const Planner: FC = () => {
 
       // check current roadmap against last-saved roadmap in local storage
       // if they are different, mark changes as unsaved to enable alert on page leave
-      dispatch(setUnsavedChanges(localStorage.getItem('roadmap') !== roadmapStr));
+      const localRoadmap = JSON.parse(localStorage.getItem('roadmap') ?? emptyRoadmap);
+      delete localRoadmap.timestamp;
+      dispatch(setUnsavedChanges(JSON.stringify(localRoadmap) !== roadmapStr));
     }
   }, [cookies, currentPlanData, dispatch, isFirstRenderer, roadmapStr, transfers]);
 
   const { unitCount, courseCount } = calculatePlannerOverviewStats();
 
   return (
     <div className="planner">
+      <Modal
+        show={showSyncModal}
+        onHide={() => {
+          setShowSyncModal(false);
+        }}
+        className="ppc-modal"
+        centered
+      >
+        <Modal.Header closeButton>
+          <h2>Cloud roadmap out-of-date</h2>
```

Comment:
Capitalization is inconsistent with other modals

<img width=300 src=https://github.com/icssc/peterportal-client/assets/54484616/022f0f6e-2b77-44b7-a049-5488a292bb37> <img width=300 src=https://github.com/icssc/peterportal-client/assets/54484616/3d4d5630-0ea6-4b40-90eb-ee58da2122ce> <img width=300 src=https://github.com/icssc/peterportal-client/assets/54484616/d79c98b8-3824-4d97-815f-3e916772aade>